Update pixel art hubs and inspiration games; remove offline Pixelation link

Brenda-Machado opened the pull request:
Hello :) 

I was working on some pixel art games and saw this repository tagged for Hacktoberfest. I thought I’d contribute by adding some useful sites I found and updating a few things.

This pull request adds links to the pixel art hubs Pixelrepo and Lospec. Additionally, I removed Pixelation as the site is no longer online. Finally, I added three games to the inspiration games list, following the ordering by release date.

Thanks for helping keep projects like this alive and supporting the pixel art community!

Looking forward to your review.

Brenda-Machado committed:
Add Pixelrepo and Lospec links and remove pixelation, not longer online

Brenda-Machado committed:
some new inspiring pixel art games

Brenda-Machado committed:
Fix order for Lospec

Siilwyn commented:
Thank you for your contribution!
I'm unsure about adding all three games, I try to have mostly awesome classics in it mixed with some new games that have a unique look. If you were to pick one or two out of these three, which one would it be and why?

:purple_heart:

Brenda-Machado commented:
Thinking about your criteria for choosing the games, I think I’d pick a classic (which I love) that has pixel art which, although simple, is quite striking: Undertale. It has an engaging narrative that made me want to start my own game, and maybe it inspired others to do the same. 

I removed the other games and added just Undertale.

Brenda-Machado committed:
Removing the extra games, addition of undertale

Siilwyn commented:
Thank you! :clap:
